tests/data/pytest/pytest.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
+; xunit1 is necessary for adding line attribute.
+junit_family = xunit1
+markers =
```

Comment:
Why do we need to add the `markers` property??

Reply:
To register markers.

https://docs.pytest.org/en/stable/example/markers.html#registering-markers

Reply:
Could leave the ULR as comment?